src/django/api/views/facility/facilities_view_set.py

```diff
@@ -224,10 +224,17 @@ def list(self, request):
             )
         sort_by = params.validated_data['sort_by']
         order_list = []
-        if (sort_by is not None) and (sort_by == 'name'):
+
+        if (sort_by is None) or (sort_by == 'contributors_desc'):
+            order_list = ['-contributors_count', 'name']
+        elif (sort_by == 'name_asc'):
             order_list = ['name']
-        else:
+        elif (sort_by == 'name_desc'):
+            order_list = ['-name']
+        elif (sort_by == 'contributors_desc'):
             order_list = ['-contributors_count', 'name']
```

Comment:
Here you use check for sort_by == 'contributors_desc' the second time. Firstly it checks at line 228.

Reply:
+1 @Innavin369

Reply:
removed